massa-protocol-worker-2/src/handlers/peer_handler/tester.rs

```diff
@@ -177,18 +185,31 @@ impl Tester {
             config.max_out_connections = 1;
 
             let mut network_manager = PeerNetManager::new(config);
-
+            println!("Tester launched");
             loop {
                 crossbeam::select! {
                     recv(receiver) -> res => {
                         match res {
                             Ok(listener) => {
+                                if let Some(peer_info) = db.read().peers.get(&listener.0) {
+                                    let timestamp = SystemTime::now()
+                                    .duration_since(UNIX_EPOCH)
+                                    .expect("Time went backward")
+                                    .as_millis();
```

Comment:
if we keep the u128 we should use .as_nano(), no ?

.as_millis() : https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.as_millis

.as_nanos() : https://doc.rust-lang.org/stable/std/time/struct.Duration.html#method.as_nanos

Reply:
I tried but had problems with the calculation of the seconds. I open a followup issue